src/main/java/com/wire/kalium/api/user/client/ClientResponse.kt

```diff
@@ -0,0 +1,37 @@
+package com.wire.kalium.api.user.client
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class ClientResponse(
+        @SerialName("id") val id: String,
+        @SerialName("cookie") val refreshToken: String,
+        @SerialName("time") val registrationTime: String,
+        @SerialName("location") val location: LocationResponse?,
+        @SerialName("address") val ipAddress: String?,
+        @SerialName("model") val model: String?,
+        @SerialName("type") val deviceType: String,
+        @SerialName("class") val deviceClass: String,
+        @SerialName("label") val label: String?
+)
+
+@Serializable
+data class LocationResponse(
+    //@SerialName("name") val name: String,
```

Comment:
Do we need name?

Reply:
he server dose not return name but there was name in the AR project so that was confusing